sanic/config.py

```diff
@@ -20,6 +26,63 @@ class Config:
 ▌     ▐                ▀▀▄▄▄▀
  ▀▀▄▄▀
 """
-    REQUEST_MAX_SIZE = 100000000  # 100 megababies
-    REQUEST_TIMEOUT = 60  # 60 seconds
-    ROUTER_CACHE_SIZE = 1024
+        self.REQUEST_MAX_SIZE = 100000000  # 100 megababies
+        self.REQUEST_TIMEOUT = 60  # 60 seconds
+        self.ROUTER_CACHE_SIZE = 1024
+
+    def __getattr__(self, attr):
+        return self[attr]
```

Comment:
according to https://docs.python.org/3/reference/datamodel.html#object.__getattr__ this method should raise `AttributeError`, but in this case it will raise `KeyError` instead